resources/languages/ko.json

```diff
@@ -2,211 +2,211 @@
+			"title": "사전 설정 삭제 메뉴",
+			"subtitle": "사전 설정을 삭제하려면 버튼을 클릭하세요"
 		},
 		"preset-list": {
 			"title": "사전 설정 목록 메뉴",
-			"subtitle": "미리보기를 편집 /보기하려면 버튼을 클릭하십시오."
+			"subtitle": "사전 설정을 편집/보려면 버튼을 클릭하세요"
 		},
 		"preset-creation": {
-			"title": "프리셋 생성 메뉴",
-			"name": "프리셋 이름",
+			"title": "사전 설정 만들기 메뉴",
+			"name": "사전 설정 이름",
 			"size": "브러시 크기",
-			"shape": "브러시 모양",
-			"type": "브러시 타입",
-			"hollow": "할로우 브러쉬",
-			"decrement": "브러쉬 제거",
+			"shape": "브러시 도형",
+			"type": "브러시 종류",
+			"hollow": "빈 브러시",
+			"decrement": "감소 브러시",
 			"height": "브러시 높이",
-			"perfect": "브러시 모양 마무리",
-			"blocks": "브러시 블럭",
-			"obsolete": "O쓸모없는 블록",
-			"biome": "쓸모없는 블록",
+			"perfect": "브러시 도형 완벽",
+			"blocks": "브러시 블록",
+			"obsolete": "쓸모없는 블록",
+			"biome": "브러시 생물군계",
 			"tree": "브러시 나무"
 		},
 		"main-menu": {
 			"title": "BlockSniper 메뉴",
-			"subtitle": "계속하려면 버튼을 클릭하십시오.",
+			"subtitle": "계속하려면 버튼을 클릭하세요.",
 			"brush": "브러시",
-			"tree": "Tree",
+			"tree": "나무",
 			"config": "구성",
-			"presets": "프리셋",
-			"global-brush": "광범위한 브러쉬",
+			"presets": "사전 설정",
+			"global-brush": "전역 브러시",
 			"exit": "나가기"
 		},
 		"brush-menu": {
-			"title": "Brush 메뉴",
+			"title": "브러시 메뉴",
 			"size": "브러시 크기",
-			"shape": "브러시 모양",
-			"type": "브러시 타입",
-			"hollow": "할로우 브러쉬",
-			"decrement": "브러쉬 제거",
+			"shape": "브러시 도형",
+			"type": "브러시 종류",
+			"hollow": "빈 브러시",
+			"decrement": "감소 브러시",
 			"height": "브러시 높이",
-			"perfect": "브러시 모양 마무리",
-			"blocks": "브러시 블럭",
-			"obsolete": "O쓸모없는 블록",
-			"biome": "쓸모없는 블록",
+			"perfect": "브러시 도형 완벽",
+			"blocks": "브러시 블록",
+			"obsolete": "쓸모없는 블록",
+			"biome": "브러시 생물군계",
 			"tree": "브러시 나무"
 		},
 		"configuration-menu": {
 			"title": "BlockSniper 구성",
-			"auto-update": "새 버전이 발견되면 자동으로 구성 파일을 업데이트합니다.",
+			"auto-update": "새로운 버전을 찾았을 때 자동으로 구성 파일 업데이트",
 			"language": "BlockSniper 관련 메시지를 보내는 데 사용되는 언어",
-			"brush-item": "브러시하는 데 사용되는 항목의 항목 ID",
-			"max-brush-size": "브러시의 절대 최대 크기",
-			"min-async-size": "비동기 브러쉬 크기 제한. 브러쉬 크기가 이보다 크면 작업이 비동기 적으로 실행됩니다",
-			"max-reverts": "저장할 Undo / Redo 저장의 최대량. 이전 버전은 자동으로 정리됩니다.",
-			"reset-decrement-brush": "감소 브러시를 초기 시작 크기로 자동 재설정",
-			"save-brush": "서버 재시작시 플레이어의 브러시 속성 저장",
-			"drop-plants": "드래프러/블로워 타입의 destrow를 제거한 후 LeafBlowertype으로 멀리 떨어져 있습니다.",
-			"session-timeout-time": "The time in minutes after which the session of a player is removed after leaving the server",
-			"auto-gui": "핫 바 슬롯을 브러시 항목을 포함하는 것으로 전환 할 때 GUI를 자동으로 엽니다.",
-			"myplot-support": "MyPlot지원을 사용할지 여부",
-			"auto-reload": "이 창을 닫을 때이 토글을 켜면 구성이 자동으로 다시로드됩니다."
+			"brush-item": "브러시에 사용되는 아이템의 ID",
+			"max-brush-size": "브러시의 절대적 최대 크기",
+			"min-async-size": "비동기 브러시 크기 제한입니다. 브러시 크기가 이보다 크다면, 작업이 비동기적으로 실행됩니다",
+			"max-reverts": "저장할 실행 취소/다시 실행 보관의 최대량입니다. 이전 것들은 자동으로 비워집니다",
+			"reset-decrement-brush": "초기 실행 크기로 감소 브러시 자동으로 초기화",
+			"save-brush": "서버 다시 시작 시에 플레이어의 브러시 속성 저장",
+			"drop-plants": "낙엽청소기 유형으로 파괴된/불어진 식물 아이템 떨어뜨리기",
```

Comment:
(English) There are two sentences that have Leaf( )Blower; one is LeafBlower (this), and the other one is Leaf Blower (below).